### DISCUSSION TITLE
gcc file locations are not shown as link and are not clickable (but are parsed into "Problems" correctly)

### DISCUSSION THREAD

**Discussion by ToBoMi** [microsoft/vscode-cmake-tools#3171] [Q&A]
I use
```
[cmake] -- The C compiler identification is GNU 11.3.1
[cmake] -- The CXX compiler identification is GNU 11.3.1
``` 
and when I introduce an issue in the program I get it parsed into the "Problems" panes properly and navigatable:

![image](https://github.com/microsoft/vscode-cmake-tools/assets/64197724/3c32f5e7-b320-4981-8664-bbc096721561)

When I look into the "output pane", I see the error in the output with relative file paths like this (altered for privacy reasons):
```
[build] [my very long and hidden build command] ../../../path/to/file.cpp
[build] In file included from ../../../path/to/file.cpp:9:
[build] ../../../path/to/file.hpp:133:52: error: expected '}' before 'sfsdf'
[build]   133 |                              "a_variable" sfsdf
[build]       |                                           ^~~~~
[build] ../../../path/to/file.hpp:131:35: note: to match this '{'
[build]   131 |         anotherVariableFoundInTheCode{
[build]       |                                   ^
[build] ../../../path/to/file.hpp:133:52: error: expected ';' before 'sfsdf'
[build]   133 |             "anotherVariableFoundInTheCode" sfsdf
[build]       |                                             ^~~~~
[build] [2/3] Building CXX object path/to/file.cpp.obj
[build] FAILED: path/to/file.cpp.obj 
```

These file paths are not shown as links and are therefor not navigatable. I expect the file links to be shown as links and navigatable.
With a MSVC compiler the paths in the error output are absolute and are linked and navigatable.

Is this configurable? Is this a bug?

**Comment by gcampbell-msft** [comment]
@ToBoMi I don't think that this is configurable in our extension. Most often we defer to the Problems pane for navigation and clicking. 

If you still are facing issues with this, please let us know and create an issue with repro projects so that we can investigate. Thanks!
